### PR TITLE
Update the permissions of /var/etcd from 0700 -> 0750, so that backup…

### PR DESCRIPTION
**Pull request by vinayakankugoyal**
… components that are part of etcd group can traverse it.

#### What type of PR is this?
/kind feature

#### What this PR does / why we need it:
This PR updates the permissions of /var/etcd from 0700 -> 0750. This change is needed so that we can run backup components as non-root, while still being able to traverse /var/etcd and read the version.txt file.

Also consolidating that logic of setting ownership of /var/etcd to mount-master-pd.

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

File: cluster/gce/gci/configure-helper.sh [hunk 1]
```diff
@@ -586,7 +586,7 @@ function mount-master-pd {
 
   # Contains all the data stored in etcd.
   mkdir -p "${mount_point}/var/etcd"
-  chmod 700 "${mount_point}/var/etcd"
+  chmod 750 "${mount_point}/var/etcd"
```
Comment (roycaihw) [inline cluster/gce/gci/configure-helper.sh:589]: I'm not the expert to tell if it's safe to allow group reading the data. The rest of the changes LGTM.

**Comment by roycaihw** [comment]
@vinayakankugoyal explained to me offline.

/lgtm

**Comment by wojtek-t** [comment]
@vinayakankugoyal - given our internal discussions and path forward, I'm assuming we no longer need this PR, right?

Can we close it?

**Pull request closed without merging**